aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -57,21 +59,23 @@ ASP.NET Core apps are hosted by IIS in a reverse proxy configuration. Some of th
 
 ## Managed modules
 
-| Module                  | .NET Core Active | ASP.NET Core Option |
-| ----------------------- | :--------------: | ------------------- |
-| AnonymousIdentification | No               | |
-| DefaultAuthentication   | No               | |
-| FileAuthorization       | No               | |
-| FormsAuthentication     | No               | [Cookie Authentication Middleware](xref:security/authentication/cookie) |
-| OutputCache             | No               | [Response Caching Middleware](xref:performance/caching/middleware) |
-| Profile                 | No               | |
-| RoleManager             | No               | |
-| ScriptModule-4.0        | No               | |
-| Session                 | No               | [Session Middleware](xref:fundamentals/app-state) |
-| UrlAuthorization        | No               | |
-| UrlMappingsModule       | No               | [URL Rewriting Middleware](xref:fundamentals/url-rewriting) |
-| UrlRoutingModule-4.0    | No               | [ASP.NET Core Identity](xref:security/authentication/identity) |
-| WindowsAuthentication   | No               | |
+Managed modules are *not* functional with hosted ASP.NET Core apps. ASP.NET Core offers middleware alternatives in several cases.
```

Comment:
Technically these managed modules are functional with the ASP.NET Core apps, although the middleware provides alternatives for a subset :)